contrib/ci/Dockerfile-arch

```diff
@@ -0,0 +1,10 @@
+FROM archlinux/base
+RUN rm /usr/share/libalpm/hooks/package-cleanup.hook
+RUN pacman -Syu --noconfirm
+RUN pacman -S --noconfirm base-devel
```

Comment:
Is it possible to install more of the build dependencies in the Dockerfile instead of the run script?  
It's not as important for Travis CI since it always builds the Dockerfile for each build, but it significantly speeds up local builds and testing to have a container with all build deps already in place.